[docs] Port multicloud section to stable (#18775)
yugabyte/yugabyte-db: docs/content/preview/develop/multi-cloud/multicloud-migration.md

type: docs
---

Moving applications from an on-premises data center to a public cloud, or from one public cloud to another, is a non-trivial task. YugabyteDB provides a simple pattern for you to accomplish this task with comparative ease.

## Migration between two clouds

To illustrate, consider a scenario where you have a 3-node cluster deployed in AWS (us-west) and would like to move to GCP (us-central).

![Multi-cloud Migration](/images/develop/multicloud/multicloud-migration-goal.png)

To accomplish the migration, use the [xCluster](../../../architecture/docdb-replication/async-replication/) feature, which provides one-way asynchronous replication from one universe to another. In this case, AWS will be the source and the GCP universe will be the target.

### Set up the universes

Set up the **Source** universe in AWS and the **Target** universe in GCP using the following setup procedures.

{{<cluster-setup-tabs>}}

When finished, note down the universe-uuids of the `source` and `target` universes.

### Bootstrap the new universe

> Now that the GCP universe has been set up, you need to populate the data from your AWS universe. This is typically referred to as **Bootstrapping**.

Reviewer: ```suggestion
After the GCP universe has been set up, you need to populate the data from your AWS universe. This is typically referred to as **Bootstrapping**.
```

{{<tip title="More Details">}}
For detailed instructions, see [Bootstrap a target universe](../../../deploy/multi-dc/async-replication/#bootstrap-a-target-universe).
{{</tip>}}

The basic flow of bootstrapping is as follows:

1. Create a checkpoint for all the tables in the AWS universe. For example:

    ```bash
    ./bin/yb-admin -master_addresses <AWS_universe_master_addresses> \
            bootstrap_cdc_producer <comma_separated_source_universe_table_ids>
    ```

> 1. Take the backup of the tables on the AWS universe and restore them to the GCP universe. See [Backup and Restore](../../../manage/backup-restore/).

Reviewer: ```suggestion
1. Take a backup of the tables on the AWS universe and restore them to the GCP universe. See [Backup and Restore](../../../manage/backup-restore/).
```

This ensures that the current data in your AWS universe is correctly copied over to the GCP universe.

### Set up replication

> Now that your data has been pre-populated from the AWS universe to the GCP universe, you need to set up the replication stream from the AWS to the GCP universe.

Reviewer: ```suggestion
After your data has been pre-populated from the AWS universe to the GCP universe, you need to set up the replication stream from the AWS to the GCP universe.
```

![Multi-cloud Replication](/images/develop/multicloud/multicloud-migration-replication.png)

> Now any data changes to the AWS universe are automatically applied to the GCP universe. _NOTE_: For now, DDL changes have to be applied manually.

Reviewer: ```suggestion
Any data changes to the AWS universe are automatically applied to the GCP universe. _NOTE_: For this example, DDL changes have to be applied manually.
```

      change_xcluster_role ACTIVE
  ```

> Now you can point your applications to the new universe in GCP and then stop the replication, as shown in the following illustration.

Reviewer: ```suggestion
You can point your applications to the new universe in GCP and then stop the replication, as shown in the following illustration.
```
